backend/CircleForms/Controllers/UsersController.cs

```diff
@@ -93,6 +95,19 @@ public async Task<IActionResult> GetMePosts()
         return Unauthorized();
     }
 
+    /// <summary>
+    ///     Get user's answers
+    /// </summary>
+    [Authorize]
+    [HttpGet(ApiEndpoints.UsersGetMeAnswers)]
+    public async Task<IActionResult> GetMeAnswers()

```

Comment:
Why would you need all answers? I'd imagine we might need per-post answers instead

Reply:
per-post answers already there. 
we might want all answers for a page that will show us all answers that we sent.